### DISCUSSION TITLE
Stellar integration

### DISCUSSION THREAD

**Discussion by irux** [Web3Auth/Web3Auth#1184] [Help]
Do someone have any kind of experience integrating with stellar? This should be possible over the Private key Provider right? What format does have the key?

Thanks!

**Comment by Saviour1001** [comment]
Hey there, I haven't interacted with Stellar Blockchain, but it looks it follows [ed25519 curve ](https://github.com/stellar/stellar-core/blob/master/src/crypto/Curve25519.h) similar to Solana. As the key provided by Web3Auth can be converted to a different curve, we can support Stellar. Have a look at our [multichain example](https://github.com/Web3Auth/examples/tree/main/web-modal-sdk/multi-chain/react-multi-chain-web3auth-example) where we are showcasing how to generate wallets for different chains at once.  
Hope this helps!
